src/test/java/net/minecraftforge/debug/DataGeneratorTest.java

```diff
@@ -201,6 +205,83 @@ protected void registerRecipes(Consumer<IFinishedRecipe> consumer)
         }
     }
 
+    public static class SoundDefinitions extends SoundDefinitionsProvider
+    {
+        public SoundDefinitions(final DataGenerator generator, final ExistingFileHelper helper)
+        {
+            super(generator, MODID, helper);
+        }
+
+        @Override
+        public void registerSounds()
+        {
+            // Taken from vanilla's 'sounds.json'. These are supposed to be 1-to-1 copies, but currently
+            // automatic testing of those copies cannot be performed due to limitations in the data-gen
+            // system, where the additional data pack that vanilla loads from the launcher's assets/
+            // directory isn't actually loaded when running data-gen.
```

Comment:
Don't we have that set up now? We should be able to compare the generated file with the original file